command.go

```diff
@@ -186,6 +186,30 @@ type Command struct {
 	versionTemplate string
 }
 
+// SetUse allows you to set the Use property after the command has been created, primarily intended to
+// facilitate localisation
+func (c *Command) SetUse(s string) {
```

Comment:
Don't we have a bunch of checks in AddCommand to make sure we never set the same command name twice? Would we need that here?